.changeset/good-ears-flow.md

```diff
@@ -0,0 +1,9 @@
+---
+"angular-workspace": major
+"@astrouxds/angular": major
+"astro-website": major
+"@astrouxds/react": major
+"@astrouxds/astro-web-components": major
+---
+
+What: Removed the title attribute from rux-icon. Why: The global HTML title attribute is not recommended for use, and was an oversight when added to rux-icon.
```

Comment:
Does this cut it for a changeset? I'm not sure what exactly to put for the 'how to migrate', since it's something that shouldn't be there at all.